flagcx/flagcx.cc

```diff
@@ -964,7 +964,7 @@ flagcxResult_t flagcxGroupStart(flagcxComm_t comm) {
 
 flagcxResult_t flagcxGroupEnd(flagcxComm_t comm) {
   if (useHeteroComm()) {
-    FLAGCXCHECK(flagcxRunners[flagcxUniRunner]->groupStart());
+    FLAGCXCHECK(flagcxRunners[flagcxUniRunner]->groupEnd());
```

Comment:
Why didn’t the check function block it? Have you considered enriching the check function?